server/test/services/export/JsonExporterTest.java

```diff
@@ -24,7 +25,9 @@ public void testAllQuestionTypesWithoutEnumerators() throws Exception {
         exporter
             .export(
                 fakeProgram.getProgramDefinition(),
-                IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG)
+                IdentifierBasedPaginationSpec.MAX_PAGE_SIZE_SPEC_LONG,
+                Optional.empty(),
+                TimeFilter.EMPTY)
```

Comment:
I'd like to see more tests in here

Reply:
The underlying implementation calls `ProgramRepository.getApplicationsForAllProgramVersions`, which already has coverage for all of the filtering ([code](https://github.com/seattle-uat/civiform/blob/5a7404a06e426cca2eda1422dbcf28b9d83c2e2e/server/test/repository/ProgramRepositoryTest.java#L180-L418)). Most of this is a pass-through.

Similar to the demography export, we do a smoke-test of the "plumbing" of the query parameters all the way through to the exported data through browser testing.

Please let me know if this doesn't work for you and I can try and find a good balance of additional tests in a subsequent PR.